app/services/daily_report.rb

```diff
@@ -5,11 +5,13 @@ def initialize(day)
     @day = day
   end
 
-  def posts(sort='')
+  def posts(sort='', new_today=false)
     range = day.beginning_of_day .. day.end_of_day
-    created_no_replies = Post.where(last_reply_id: nil, created_at: range)
+    created_today = Post.where(created_at: range)
+    return created_today.select("posts.*, posts.created_at as first_updated_at").order(sort) if new_today
```

Comment:
Why are you aliasing that?

Reply:
...ah, because the select is also pulling replies.created_at later, cool

Reply:
(also default sort uses first_updated_at)